github/teams_discussions.go

```diff
@@ -44,12 +44,12 @@ type DiscussionListOptions struct {
 	Direction string `url:"direction,omitempty"`
 }
 
-// ListDiscussions lists all discussions on team's page.
+// ListDiscussionsByID lists all discussions on team's page given Organization and Team ID.
 // Authenticated user must grant read:discussion scope.
 //
 // GitHub API docs: https://developer.github.com/v3/teams/discussions/#list-discussions
-func (s *TeamsService) ListDiscussions(ctx context.Context, teamID int64, options *DiscussionListOptions) ([]*TeamDiscussion, *Response, error) {
-	u := fmt.Sprintf("teams/%v/discussions", teamID)
+func (s *TeamsService) ListDiscussionsByID(ctx context.Context, orgID, teamID int64, options *DiscussionListOptions) ([]*TeamDiscussion, *Response, error) {
```

Comment:
Let's please change `options` to `opts` to match what was recently done in #1417.
Here and throughout this PR.